setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+all_args=$@
+echo $all_args > /tmp/test_file11
+bucket=hailstone-ami-scan-results
+cd /usr/local/bin
+PATH=${PATH}:/usr/local/bin
+echo "after path setup"
+#sleep
+yum install -y epel-release wget
+export SCAP_TARGET='stig-rhel7-disa'
+export TAILORING_SUFFIX='_basefile'
+wget https://raw.githubusercontent.com/CivicActions/drydock/master/baselines/baseline7/scripts/build-yum
+#wget https://raw.githubusercontent.com/CivicActions/drydock/master/baselines/baseline7/scripts/openscap-harden
+chmod u+x ./build-yum
+
+#installing openscap
+yum install -y openscap-utils scap-security-guide
+
+#install python if not exist
+cd ~
+rpm -aq python && true || yum install python
+python -m pip && true || wget https://bootstrap.pypa.io/get-pip.py; python get-pip.py --user
+python -m pip install awscli --user
+export PATH=~/.local/bin/:$PATH
+
+[ -f "/etc/centos-release" ] && OS=centos7 || OS=rhel7
+
+#scaning rhel7-disa
```

Comment:
I would suggest we use log messages rather than code comments and reserve the latter for explanatory content.